src/tests/resistance/random_bytes.rs

```diff
@@ -0,0 +1,43 @@
+use std::time::Duration;
+
+use tempfile::TempDir;
+use tokio::time::sleep;
+
+use crate::{
+    fuzzing::{random_bytes, seeded_rng},
+    setup::node::{Node, NodeType},
+    tools::synth_node::SyntheticNode,
+};
+
+const ITERATIONS: usize = 20;
+const DISCONNECT_TIMEOUT: Duration = Duration::from_secs(2);
+
+#[tokio::test]
+async fn r002_node_must_disconnect_when_receiving_random_bytes() {
+    // ZG-RESISTANCE-002
+
+    let mut rng = seeded_rng();
+    let payloads = random_bytes(&mut rng, ITERATIONS);
+
+    let target = TempDir::new().expect("Can't build tmp dir");
+
+    let mut node = Node::builder()
+        .start(target.path(), NodeType::Stateless)
+        .await
+        .unwrap();
```

Comment:
Let's rather use `expect` instead of `unwrap` throughout our codebase (unless `unwrap` is garnatueed to succeed)

Reply:
done